not-a-toy.R

```diff
@@ -164,7 +159,7 @@ for(i in unique(incdat$id)) {
     # Estimate starting k by slope of 13C.  This follows para. 21:
     # "We then calculate k as the slope of the linear regression of ln(n)
     # versus time...
-    m <- lm(log(cal13CH4ml + 1/FRAC_K) ~ time_days, data = dat)
+    m <- lm(log(cal13CH4ml) ~ time_days, data = dat)
```

Comment:
I think we're right about this (that the fractionation is NOT within the log function), but might be worth discussing with Joe